.github/workflows/push.yml

```diff
@@ -19,8 +19,9 @@ jobs:
           - version: 10.x
           - version: 12.x
           - version: 14.x
+          - version: 15.x
```

Comment:
Note this will fail for a few days until we bump the major version on core, but that's fine since those tests are allowed to fail anyway.